src/mpc-hc/AppSettings.h

```diff
@@ -420,6 +420,32 @@ class CUIceClient : public CRemoteCtrlClient
 
 #define APPSETTINGS_VERSION 8
 
+class RecentFileEntry {
+public:
+    RecentFileEntry() {}
+    RecentFileEntry(const RecentFileEntry &r) {
+        cue = r.cue;
+        title = r.title;
+        fns.Copy(r.fns);
+        subs.Copy(r.subs);
+    }
+
+    CString title;
+    CAtlArray<CString> fns;

```

Comment:
Maybe use List instead of Array? Other places in the code also use List. That could simplify code because you don't need to copy data from a List to an Array or the other way around.